src/Admin/AdminInterface.php

```diff
@@ -72,6 +72,9 @@ public function attachAdminClass(FieldDescriptionInterface $fieldDescription): v
 
     public function getDatagrid(): DatagridInterface;
 
+    // NEXT_MAJOR: uncomment this method in 4.0
+    //public function getPagerType(): string;
```

Comment:
You lost the @method and didn't uncomment this.

Reply:
I never apply the `NEXT_MAJOR` on merge, and thought the person who would would fix the `@method` issue, so to me it wasn't required.

Reply:
> I never apply the `NEXT_MAJOR` on merge, and thought the person who would would fix the `@method` issue, so to me it wasn't required.

The person who introduced this was me. That's why I made the change on my PR 😅
I'll made this after your PR then.